cobbler.spec

```diff
@@ -424,6 +406,7 @@ fi
 %config(noreplace) %{_sysconfdir}/cobbler/settings.d/bind_manage_ipmi.settings
 %config(noreplace) %{_sysconfdir}/cobbler/settings.d/manage_genders.settings
 %config(noreplace) %{_sysconfdir}/cobbler/settings.d/nsupdate.settings
+%config(noreplace) %{_sysconfdir}/cobbler/settings.d/windows.settings
```

Comment:
All newly added settings should be documented and included in the schema validation. Otherwise Cobbler won't start.